src/routes/NewUser/components/NewUserPage/NewUserPage.js

```diff
@@ -94,12 +95,67 @@ function NewUser() {
     errors,
     register,
     formState: { isValid, dirty },
+    getValues,
+    setValue,
   } = useForm({
     validationSchema: userProfileSchema,
     defaultValues,
   });
   const [userLocationInfo, setUserLocationInfo] = useState(null);
 
+  /**
+   * Sets the value for the given field if it isn't already populated.
+   * @param {string} fieldName name of the field which will be set
+   * @param {string} allFieldValues current value of all fields
+   * @param {object} googleResultComponent address component returned by the Google Maps API
+   * @param {object} addressType type of address component to lookup
+   * @param {string} addressTypeFormat short_name or long_name
+   */
+  function setValueIfNotAlreadySet(
+    fieldName,
+    allFieldValues,
+    googleResultComponent,
+    addressType,
+    addressTypeFormat = 'long_name',
+  ) {
+    if (
+      googleResultComponent.types.indexOf(addressType) !== -1 &&
```

Comment:
We can use includes like so: `googleResultComponent.types.includes(addressType)`

Reply:
You are going to make me switch this behavior! Updated. 😀